ibm/mas_devops/roles/cp4d_db2wh_backup/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+# db2 backup default vars
+db2wh_backup_dir: "{{ lookup('env', 'DB2WH_BACKUP_DIR') }}"
+cpd_meta_namespace_source: "{{ lookup('env', 'CPD_NAMESPACE_SOURCE') }}"
```

Comment:
Will this work with cpd v3 and v4 .. in v4 the db2 pod is not in the meta namespace, but a seperate services namespace?  e.g. could we do with some code like this: https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_db2wh/tasks/main.yml#L7-L45 somewhere in this role?

Reply:
There's no backup/restore process change whether this is a v3.5 or v4.0, it all behaves the same no matter cpd version, during tests I even used a source db2wh from v3.5 and restored in a target db2wh v4.0 to ensure, and it works fine either way.

Reply:
fixed to be same as in `cp4d_db2wh` and `cp4d_db2wh_manage_hack` as you suggested.